base.py

```diff
@@ -0,0 +1,124 @@
+import heapq
+import random
+import jsoneng
+from dotenv import load_dotenv
+import os
+
+jdb = jsoneng.JsonDB()
+jdb.create({})
+
+import random
+import heapq
+
+class Character:
+    def __init__(self, name):
+        self.name = name
+        self.state = "idling"
+
+    def __repr__(self):
+        return self.name
+
+class Event:
+    def __init__(self, time, action, participants=None):
+        self.time = time
+        self.action = action
+        self.participants = participants if participants is not None else []
```

Comment:
Enum for actions perhaps?

Reply:
it should be location actually but oh this is actually not the right file

but in the future we'll do classes for those as enums, dataclasses instead